src/lib/profile.js

```diff
@@ -0,0 +1,68 @@
+import { getProfile, getVerifiedAccounts, openBox } from '3box'
```

Comment:
I was kind of hesitant to call it `profile-utils` since we perform api calls on it. What i was thinking we could do is separate the `lib` folder from the `utils` folder and move there everything that is `-utils`. 
What do you think?

Reply:
yeah that could be an option!

Reply:
Cool, created an issue for that https://github.com/1Hive/honey-pot/issues/74